schedule.html

```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
           </tr>
         </tbody>
       </table>
+      <p>
+        (This schedule may change based on discussions with the <a href=
+        "./#program">program committee</a>.)
+      </p>
+      <h2 id="instructions">
+        Instructions for participants
+      </h2>
+      <p>
+        The workshop will focus around several topics identified by the
+        <a href="https://www.w3.org/2002/09/wbs/1/vr-workshop/results">expressions
+        of interest</a> and <a href="papers.html">position statements</a>. Each
+        topic will be introduced by one or more related <a href=
+        "#lightning">lightning talks</a>, and will be explored more in-depth in
+        <a href="#breakout">breakout sessions</a>, concluded with joint
+        summaries. The goal of each discussion at the workshop is not to
+        resolve the technical issues of the topic, but to determine its
+        relevance and priority to standardization.
+      </p>
+      <h3 id="format">
+        Format
+      </h3>
+      <dl class="format">
+        <dt>
+          <span class="plenary" id="plenary"></span>
+        </dt>
+        <dd>
+          Plenary sessions introduce and summarize information relevant to all
+          participants.
+        </dd>
+        <dt>
+          <span class="lightning" id="lightning"></span>
+        </dt>
+        <dd>
+          Discussion topic (30-60 mins) starts off with 3-4 5-minute <a href=
+          "https://en.wikipedia.org/wiki/Lightning_talk">lightning talks</a> by
+          speakers nominated in advance. Lightning talks are followed by group
```

Comment:
this is mentioned already [above](https://github.com/w3c/vr-workshop/pull/40/files#diff-0953deb6637fd0edbb59e4e408606954R371), but I suppose it's fine to repeat it so people don't miss